### ISSUE TITLE
setErrorHandlerForCode to ignore an error?

### ISSUE THREAD

**Issue by StephenDaedalus** [danthorpe/Operations#215]
I'm testing one my CloudKitOperation and am getting a .PartialFailure that I want to ignore. Do you have a recommendation as to how this should this be handled? 

Should I use a custom handler and setErrorHandlerForCode is there a simpler way to intercept and simply swallow the error?

**Comment by danthorpe** [comment]
Hmm - well, `CloudKitOperation` doesn't have any automatic "handling" of `.PartialFailure`. But it's one of the errors that you're almost certainly gonna want to process in a production quality app.

So... you could try this:

```swift
operation.setErrorHandlerForCode(.PartialFailure) { _, log, _ in
    log.info("Ignoring partial error.")
    return .None
}
```

Which essentially is going to provide an error handler which does nothing.


Perhaps it would be good if `RetryOperation` had some functionality where it would be possible to determine whether or not it should add a new operation in the event of finishing with errors. Something like:

```swift
operation.ignoreErrors: ([ErrorType] -> Bool)? = .None
```

**Comment by danthorpe** [comment]
Hi @StephenDaedalus  - did this work?

**Comment by StephenDaedalus** [comment]
I closed it to avoid you having to hassle with my request. Compared with the finished(errors: [ErrorType]) issue mine seemed trivial.

It makes sense that CloudKitOperation knows about how to recover from certain error codes. I thought there may be some logic to intercept error codes. 

I've been able to work through it so I closed it. 

Note I also closed my other open issue. I made a change to my use of the framework to use ComposedOperation. Note that I had to make the "init" public or else it wouldn't compile.
